interface/wx/webview.h

```diff
@@ -460,11 +460,45 @@ class wxWebView : public wxControl
     virtual void Reload(wxWebViewReloadFlags flags = wxWEBVIEW_RELOAD_DEFAULT) = 0;
 
     /**
-        Runs the given javascript code.
-        @note When using wxWEBVIEW_BACKEND_IE you must wait for the current
+        Sets IE emulation level.
+        @param defaultLevel @false to set level to IE7, @true to set level to old mode.
+        @return @true if level can be set, @false if it is not.
+        @note Only avaliable on wxWEBVIEW_BACKEND_IE.
+        @since 3.1.1
+    */
+    virtual bool MSWSetEmulationLevel(bool defaultLevel = false) = 0;
```

Comment:
This documentation is out of date, it's not virtual any more (and the `@note` above is wrong too).